ietf/templates/meeting/session_details_panel.html

```diff
@@ -320,7 +320,7 @@ <h3 class="mt-4">Notes and recordings</h3>
                 </tr>
             {% endif %}
             {# Recordings #}
-            {% if meeting.number|add:"0" >= 80 %}
+            {% if meeting.number|add:"0" >= 80  or "interim" in meeting.number %}
```

Comment:
```suggestion
            {% if meeting.type_id == 'ietf' and meeting.number|add:"0" >= 80  or meeting.type_id != 'ietf' %}
```

Reply:
The above suggestion is a bit awkward - it requires knowing the precedence of operators as documented at https://docs.djangoproject.com/en/5.0/ref/templates/builtins/#boolean-operators. It might be better to move this into the view code to get away from the `|add:"0"` type-coersion-by-hack-trick anyhow.

Reply:
I thought of something like that.  In addition to the one mentioned above, the other [tests against meeting 60](https://github.com/ietf-tools/datatracker/blob/main/ietf/templates/meeting/session_buttons_include.html#L129).  What's the significance of meetings 60 and 80?  I'd like to remove the magic numbers, replacing with a reasonable function name.

Reply:
they aren't magic (though they aren't complete) - they are places where the _shape_ of proceedings and what was captured at meetings fundamentally changes.  60 was when we first started capturing chatlogs (jabber at the time). 80 was a point at where recording tech changed significantly.

That said, we don't have any Documents of type "recording" before ietf-100, so the code being touched here could could (for meetings of type ietf) not show the recordings block at all for meeting numbers < 100.